README.md

```diff
@@ -1,24 +1,167 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
 
-Things you may want to cover:
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
-* Ruby version
+OPTIONAL SECTIONS:
+- FAQ
+-->
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 hello-rails-react <a name="about-project"></a>
 
-* How to run the test suite
+**hello-rails-react** is a project to implement a connection between a Ruby on Rails back-end and React front-end in the same repository.
+
+## 🛠 Built With <a name="built-with"></a>
+- `Ruby on Rails`
+- `React`
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[RoR backend with an API endpoint]**
+- **[React front end]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- Ruby version 3.1.3 or later
+- Rails version 7 or later
+- Postgresql
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git https://github.com/AlejandraKeber/hello-rails-react.git
+  cd hello-rails-react
+```
+
+### Install
+
+Install this project with:
+
+  `bundle install`
+
+### Usage
+
+To start the server, execute the following command:
+
+`./bin/dev`
+
```

Comment:
- [ ] [OPTIONAL] It will also be nice to add the command to run the migrations and load the data in the seed file to your README. This will make your project more professional.
``` ruby
rails db:migrate

rails db:seed
```